Voicemail instructions for voip.ms (#106)
hyphacoop/handbook: guides.md

### Accessing Voicemail 

We use a VoIP phone line provider with forwarding and voicemail from [voip.md](https://www.voip.ms). It helps us:

- Have a phone number without tying it to a physical location
- Receive voicemail by email to [members@hypha.coop](mailto:members@hypha.coop)
- Receive text messages (SMS) to common e-mail

To access voicemail inbox you can either call remotely or through a configured SIP client.

- Remotely

>   1. Call the hypha number `4378876936`  

Reviewer: ```suggestion
  1. Call the number `4378876936`  
```

>   1. Wait for the intro audio to finish and then a short 1 secon pause

Reviewer: ```suggestion
  1. Wait for the intro audio to finish followed by a 1 second pause
```
